app/serializers/user_serializer.rb

```diff
@@ -43,10 +23,6 @@ def max_subjects
     @model.subject_limit
   end
 
-  def uploaded_subjects_count
```

Comment:
Nevermind, you're just removing useless wrappers around an attribute that Restpack can find on it's own.